platform/viewer/src/api/baseUrls.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  studies: 'url',
```

Comment:
Any reason to add this data here? Can it be empty?

Reply:
This is just an initial file for us to keep in mind that we should centralize API URLs there.
I'll remove the `studies` key/value and keep this file empty for future use :)
Thanks!